packages/typescript/index.js

```diff
@@ -26,17 +30,26 @@ function typescript(options = {}) {
               use: [
                 {
                   loader: 'awesome-typescript-loader',
-                  options
+                  options: Object.assign(
+                    {
+                      useCache: true,
+                      cacheDirectory: 'node_modules/.awcache',
+                      useBabel: true,
```

Comment:
Let's remove `babel`-related settings from the defaults.

Reply:
`useBabel` should be remove as well

Reply:
Is it a given that typescript block users are aware that they will need to pass these settings themselves if they use JSX? Or is it documented somewhere already?

Reply:
Never mind, I just saw that the sample app contains a .tsx file anyway and the test passes. So I guess there's simply something wrong with the webpack blocks config in my own app.

Reply:
TypeScript supports JSX out of the box.